tests/integration/example/environment/mixerEnvoyEnv/mixer_envoy_env.go

```diff
@@ -60,10 +62,20 @@ func (mixerEnvoyEnv *MixerEnvoyEnv) GetComponents() []framework.Component {
 
 	// Define what components this environment has
 	comps := []framework.Component{}
-	comps = append(comps, fortioServer.NewLocalComponent("my_fortio_server", mixerEnvoyEnv.TmpDir))
-	comps = append(comps, mixer.NewLocalComponent("my_local_mixer", mixerEnvoyEnv.TmpDir, mixerEnvoyEnv.configDir))
-	comps = append(comps, proxy.NewLocalComponent("my_local_envoy", mixerEnvoyEnv.TmpDir))
+	comps = append(comps, fortioServer.NewLocalComponent("my_fortio_server",
+		fortioServer.LocalCompConfig{
+			LogFile: fmt.Sprintf("%s/%s.log", mixerEnvoyEnv.TmpDir, "my_local_fortio"),
```

Comment:
This looks cumbersome.
I think logFile can be a built in concept, instead of having to add it to every Config of components.

Reply:
I prefer leave it inside the config:
1. Not every component requires a log file
2. I think it would be great to generalize concepts and `config`, `status` is a better way to combine fields and share them inside/outside of a component